mllam_data_prep/config.py

```diff
@@ -120,7 +137,8 @@ class InputDataset:
         1) the path to the dataset,
         2) the expected dimensions of the dataset,
         3) the variables to select from the dataset (and optionally subsection
-           along the coordinates for each variable) and finally
+           along the coordinates for each variable) and/or the variables to derive
```

Comment:
Hmm I haven't really thought about if they should be mutually exclusive or if we should be able to have both in the same named dataset under `inputs`. But at the moment we can't since I have not added any feature of merging the subset from `variables` with the subset from `derived_variables`.

If we want to have them mutually exclusive then `mllam_data_prep/config.py` would need to reflect that as well, as you have observed @mafdmi. Then in `mllam_data_prep/derived_variables.py` line 154 should be an `elif` and not an `if` I guess,:
```diff
- if derived_variables:
+ elif derived_variables:
```

If we don't want to have them mutually exclusive, we could perhaps try and merge them after having derived them. But this will then require some additional development.

Thoughts @leifdenby ?

Reply:
I have removed the 'and'.